processing/src/main/java/io/druid/segment/data/CompressedVSizeIntsIndexedSupplier.java

```diff
@@ -312,6 +324,12 @@ protected int _get(int index)
       // removes the need for padding
       return buffer.get(buffer.position() + index) & 0xFF;
     }
+
+    @Override
+    public void inspectRuntimeShape(RuntimeShapeInspector inspector)
+    {
+      inspector.visit("buffer", buffer);
```

Comment:
As I see this is a mutable field. Could have "null", then something else, but description will be 'null' for all the instances, right?
Also 'bigEndian' is changed during processing when loadBuffer() is called.